orion/pkg/repos/user_repo_test.go

```diff
@@ -0,0 +1,194 @@
+package repos_test
+
+import (
+	"database/sql"
+	"github.com/DATA-DOG/go-sqlmock"
+	"github.com/ahsu1230/mathnavigatorSite/orion/pkg/domains"
+	"github.com/ahsu1230/mathnavigatorSite/orion/pkg/repos"
+	"reflect"
+	"testing"
+	"time"
+)
+
+func initUserTest(t *testing.T) (*sql.DB, sqlmock.Sqlmock, repos.UserRepoInterface) {
+	db, mock, err := sqlmock.New()
+	if err != nil {
+		t.Fatalf("an error '%s' was not expected when opening a stub database connection", err)
+	}
+	repo := repos.CreateTestUserRepo(db)
+	return db, mock, repo
+}
+
+//
+// Test Select All
+//
+func TestSelectAllUsers(t *testing.T) {
+	db, mock, repo := initUserTest(t)
+	defer db.Close()
+
+	// Mock DB statements and execute
+	now := time.Now().UTC()
+	rows := sqlmock.NewRows([]string{"Id", "CreatedAt", "UpdatedAt", "DeletedAt", "FirstName", "LastName", "MiddleName", "Email", "Phone", "IsGuardian", "GuardianId"}).
+		AddRow(1, now, now, sql.NullTime{}, "John", "Smith", "Middle", "john.smith@example.com", "555-555-0100", false, 0)
```

Comment:
since this table has a more columns, can you make this more "vertical"?
```
sqlmock.NewRows([]string{
   "Id",
   "CreatedAt",
   ...
}).
AddRow(
   1, 
    ...
)
```